src/main/java/com/github/manolo8/darkbot/extensions/features/FeatureDefinition.java

```diff
@@ -162,4 +163,26 @@ public boolean canLoad() {
     public void addIssue(@NotNull String message, @NotNull String description, @NotNull Level level) {
         issues.add(message, description, PluginIssue.Level.values()[level.ordinal()]);
     }
+
+    /**
+     * @return true if exception cause was {@link FeatureException}
+     */
+    public boolean addIssue(Throwable e, String message, PluginIssue.Level level) {
+        FeatureException featureException = findFeatureException(e);
+        if (featureException != null) {
+            String description = featureException.getCause() == null
+                    ? featureException.getDescription() : IssueHandler.createDescription(featureException.getCause());
```

Comment:
```suggestion
            String description = IssueHandler.createDescription(featureException);
```
Make IssueHandler.createDescription itself be the one who can provide a different/simpler description if the passed exception instanceof FeatureException.